lib/exchange/clientCredentials.js

```diff
@@ -100,11 +101,11 @@ module.exports = function(options, issue) {
       }
       if (!Array.isArray(scope)) { scope = [ scope ]; }
     }
-    
+
     function issued(err, accessToken, refreshToken, params) {
       if (err) { return next(err); }
       if (!accessToken) {
-        return next(new TokenError('Invalid client credentials', 'invalid_grant'));
+        return next(new TokenError(g.f('Invalid client credentials', 'invalid_grant')));
```

Comment:
No, closing bracket in wrong place.
`g.f('Invalid client credentials'), 'invalid_grant'`